components/script/dom/document.rs

```diff
@@ -1710,6 +1711,19 @@ impl Document {
 
         // Step 12.
         // TODO: completely loaded.
+
+        // Step 13.
+        let htmlelement = self.get_html_element();
+        let result = Microdata::parse(self, htmlelement.unwrap().upcast::<Node>());
+        if !result.get("vcard").unwrap().is_empty() {
+            let event = ScriptMsg::SendMicrodata(result.get("vcard").unwrap().to_string(), "vcard".to_string());
+            self.send_to_constellation(event);
+            self.SetTitle(DOMString::from("Extracted vCard".to_string()));
```

Comment:
We should not be setting the title in this code here. That should happen in the embedder instead.